backend/tcs-service-test/Services/ReportBusinessLogic.test.cs

```diff
@@ -30,15 +31,149 @@ public void Dispose()
+        public void PeakHours_FiveSessionsAt10AM_TwoSessionsAt5PM()
+        {
+            var sessions = new List<Session>();
+
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 24, 10, 15, 0), OutTime = new DateTime(2020, 12, 24, 10, 35, 0) });
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 24, 10, 30, 0), OutTime = new DateTime(2020, 12, 24, 10, 35, 0) });
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 25, 10, 30, 0), OutTime = new DateTime(2020, 12, 25, 10, 45, 0) });
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 26, 10, 45, 0), OutTime = new DateTime(2020, 12, 23, 10, 55, 0) });
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 27, 10, 30, 0), OutTime = new DateTime(2020, 12, 23, 10, 35, 0) });
+
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 27, 17, 30, 0), OutTime = new DateTime(2020, 12, 23, 17, 35, 0) });
+            sessions.Add(new Session() { InTime = new DateTime(2020, 12, 30, 17, 30, 0), OutTime = new DateTime(2020, 12, 30, 17, 35, 0) });
+
+            var results = ReportsBusinessLogic.PeakHours(sessions, new DateTime(2020, 12, 24), new DateTime(2020, 12, 30));
+            Assert.Equal(5, results.Where(x => x.Hour == "10 A.M").FirstOrDefault().Count);
+            Assert.Equal(2, results.Where(x => x.Hour == "5 P.M").FirstOrDefault().Count);
+        }
+
+        [Fact]
+        public void ClassTours_OneSchoolWithTwoTours_OneSchoolWithOneTour_SummedCorrectly()
+        {
+            var tours = new List<ClassTour>();
+
+            tours.Add(new ClassTour() { DayVisited = new DateTime(2020, 12, 24), Name = "Lincoln High School", NumberOfStudents = 25 });
+            tours.Add(new ClassTour() { DayVisited = new DateTime(2020, 12, 28), Name = "Lincoln High School", NumberOfStudents = 20 });
+            tours.Add(new ClassTour() { DayVisited = new DateTime(2020, 12, 30), Name = "Larry David High School", NumberOfStudents = 35 });
+
+            var results = ReportsBusinessLogic.ClassTours(tours, new DateTime(2020, 12, 24), new DateTime(2020, 12, 30));
+            Assert.Equal(45, results.Where(x => x.Name == "Lincoln High School").FirstOrDefault().Students);
+            Assert.Equal(35, results.Where(x => x.Name == "Larry David High School").FirstOrDefault().Students);
+        }
+
+        [Fact]
+        public void Volunteers_OneTeacherWithTwoSessions_OtherTeacherWithOneSession()
+        {
+            var sessions = new List<Session>();
+            var people = new List<Person>();
+            people.Add(new Person() { PersonType = PersonType.Teacher, Email = "teacher1@wvup.edu", FirstName = "Teacher", LastName = "One", Id = 12345 });
+            people.Add(new Person() { PersonType = PersonType.Teacher, Email = "teacher2@wvup.edu", FirstName = "Teacher", LastName = "Two", Id = 45678 });
+            db.People.AddRange(people);
+            db.SaveChanges();
```

Comment:
You shouldn't need to save to the database in any of these tests.
I think what you want to do here is contruct the Session with the Person constructed on the session.
EX: 
``` 
new Sesssion(){
  Person: new Person() {
    Email: ""
  }
}
```